tests/ui-cargo/module_style/fail_mod_remap/src/main.rs

```diff
@@ -0,0 +1,7 @@
+// compile-flags: --remap-path-prefix /=/remapped
```

Comment:
I assume this will not work correctly on windows, not really sure what to do about it...

Reply:
Found a solution in rust's testsuite.